amundsen_application/static/js/components/common/Badges/styles.scss

```diff
@@ -0,0 +1,12 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+@import 'variables';
+
+.clickable-badge {
+  .flag {
+    &:hover {
+      cursor: pointer;
+    }
+  }
```

Comment:
nit: On second thought I should have suggested this instead. 
```
clickable-badge {
    &:hover {
      cursor: pointer;
    }
}
```
The reason is that the. pointer cursor should appear when hovering over the element that triggers an interaction, and based on your markup, the element that the click event is attached to is the `clickable-badge` itself.